src/DynamoCoreWpf/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="DynamoVisualProgramming.LibG_228_0_0" version="2.12.0.6565" targetFramework="net48" developmentDependency="true" />
+  <package id="Greg" version="2.1.7971.20260" targetFramework="net48" />
```

Comment:
Shouldn't this be version `2.1.7972.24350` as above?

Reply:
yes thanks!